readme.txt

```diff
@@ -2,8 +2,8 @@
 Contributors: equalizedigital, alh0319, stevejonesdev
 Tags: accessibility, accessible, wcag, ada, WP accessibility
 Requires at least: 6.2
-Tested up to: 6.5.5
-Stable tag: 1.14.3
+Tested up to: 6.6
```

Comment:
Should this be semantic versioning?

```suggestion
Tested up to: 6.6.0
```

Reply:
@SteveJonesDev I updated this with the 3 digit but I remember looking at the parser for readmes a while back and I think it ignores the last digit anyways. No harm having it but I don't think it checks it when doing the routine that shows the 'not tested' check.

Reply:
I looked at other plugins and I can confirm it updates the patch versions automatically.